tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "experimentalDecorators": true,
```

Comment:
Could be over the `tsconfig.json` from `js-quic`. It already has `experimentalDecorators`.